client/me/account/main.jsx

```diff
@@ -17,7 +17,7 @@ import LanguagePicker from 'calypso/components/language-picker';
 import SectionHeader from 'calypso/components/section-header';
 import MeSidebarNavigation from 'calypso/me/sidebar-navigation';
 import { protectForm } from 'calypso/lib/protect-form';
-import config from '@automattic/calypso-config';
+import config, { isEnabled } from '@automattic/calypso-config';
```

Comment:
Instead of the config we need this check https://github.com/Automattic/wp-calypso/blob/1d2278647dc980ae462ef6a472464d628d4a6930/client/blocks/site/index.jsx#L22